Adyen/UI/Form/Items/Value Pickers/Phone Extension Picker/FormPhoneExtensionPickerItemView.swift

```diff
@@ -4,25 +4,78 @@
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
 //
 
+import Foundation
 import UIKit
 
-internal final class FormPhoneExtensionPickerItemView: BaseFormPickerItemView<PhoneExtension> {
-
-    private lazy var phoneExtensionInputControl = PhoneExtensionInputControl(inputView: pickerView,
-                                                                             inputAccessoryView: pickerViewToolbar,
-                                                                             style: item.style.text)
-
-    override internal func createInputControl() -> PickerTextInputControl {
-        phoneExtensionInputControl
+/// A control to select a phone extension from a list.
+@_spi(AdyenInternal)
+public final class FormPhoneExtensionPickerItemView: FormItemView<FormPhoneExtensionPickerItem> {
+    
+    internal lazy var valueLabel = UILabel(style: item.style.text)
+    
+    internal var accessoryImage: UIImage? { UIImage(
+        named: "chevron_down",
```

Comment:
Only suggestion is to move this to a constant? Only if you want, not a big deal

Reply:
Would make this part of a bigger refactor of moving all `UIImage(named:)` usages to a central place